src/crossfilter.js

```diff
@@ -121,6 +138,11 @@ function crossfilter() {
     offset = tmp.offset;
     one = tmp.one;
     zero = ~one;
+      
+    // Expose the dimension mask for internal use
+    dimension._offset = offset;
```

Comment:
@gordonwoodhull @tannerlinsley @esjewett Ok, I updated the push request to remove exposing dimension._offset/_zero/_one and replace it with a cleaner dimension ID.